README.md

```diff
@@ -74,3 +114,33 @@ redis.connect(host: "localhost", port: 6379) { (redisError: NSError?) in
     }
 }
 ```
+
+Next, build the program and run it (either within Xcode or on the command line):
+```
+$ swift build
+$ .build/debug/redisExample
+```
+
+You should see:
+```
+$ Connected to Redis
+$ Redis on Swift
+```
+This shows that we've connected to Redis, set a string value for a key and then successfully retrieved the value for that key.
+
+## Run Tests
```

Comment:
It feels a bit odd to encounter this section here.  Maybe it's the title?  I might prefer something like this:
## Contributing
Contributions to the Kitura-redis project are welcome.  You will want to be able to test your changes locally before submitting a pull request.

The tests require a Redis server accessible locally on the default port (6379).  If you do not wish to install Redis permanently, you can use Docker to run a temporary instance locally as follows:
```
docker run -d -p 6379:6379 redis:alpine redis-server --requirepass password123
```
The password specified above must match the one defined in `Tests/SwiftRedis/password.txt`.  Then you can run the tests as normal, either via Xcode or with:
```
swift test
```